float64[] ChildPivotPt
float64 erp
float64 maxForce
---
bool success
string info
